deepchecks/core/check.py

```diff
@@ -206,6 +212,79 @@ def _display_to_json(self) -> List[Tuple[str, str]]:
         matplotlib.use(old_backend)
         return displays
 
+    def to_wandb(self, wandb_init: bool = True, **kwargs: Any):
```

Comment:
It's a bit confusing that if wandb_init = True then there are args we *must* give, such as project. 
Not sure how to solve because it's indeed not mandatory if wandb_init = False...